source/examples/examples/tutorials.md

```diff
@@ -15,6 +15,13 @@ First we'll test our "Login Page", refactor, and then create a {% url "Custom Co
 | -- | -- | -- | -- |
 | {% url "Organizing Tests, Logging In, Controlling State" https://www.youtube.com/watch?v=5XQOK0v_YRE %} | {% fa fa-video-camera %} 27:21 | <time datetime="2018-02-22T16:00:00.000Z">02-22-2018</time> | 2.1.0 |
 
+# Video playlists
```

Comment:
This seems out of place - the intro section says "In this tuturial we will" indicating this is a single tutorial to follow and then this section comes up in between this and the start of the tutorial.